src/test/java/ca/qaguru/oranghrmbatch27/library/TestBase.java

```diff
@@ -13,15 +13,17 @@ public abstract class TestBase {
     //protected  ChromeDriver driver;
     protected String browser;
     protected final String BASE_URL = "https://opensource-demo.orangehrmlive.com";
+
     @BeforeSuite
-    public void beforeSuite(){
+    public void beforeSuite() {
         WebDriverManager.chromedriver().setup();
         WebDriverManager.firefoxdriver().setup();
+        beforeMethod("chrome");
```

Comment:
Running all the testcases within a single browser window to save time
how we can consider other browser
a mechanism to relaunch the browser if it crashes in between
-- can be kept in a separate pull request to revisit the idea

For now use the existing TestBase approach so that it can be merged to main

Reply:
Made necessary changes to run the test with the existing TestBase approach.
please review to merge to main.